tests/modules/test_api.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of RERO ILS.
+# Copyright (C) 2017 RERO.
+#
+# RERO ILS is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# RERO ILS is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with RERO ILS; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, RERO does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Utils tests."""
+
+
+from rero_ils.modules.documents.api import Document
+
+
+def test_api_update_replace(db, minimal_document_record):
+    """Test api replace."""
+    minimal_document_record['publicationYear'] = 1999
+    doc = Document.create(minimal_document_record)
+    del minimal_document_record['publicationYear']
+    doc.update(minimal_document_record, dbcommit=True)
+    assert doc.get('publicationYear') == 1999
+    doc.replace(minimal_document_record, dbcommit=True)
+    assert doc.get('publicationYear', True)
```

Comment:
Need to change this assert to something like:
assert not doc.get('publicationYear')

Reply:
changed to not in